src/app/Dashboard/dashboard-utils.tsx

```diff
@@ -394,6 +396,7 @@ export interface DashboardCardDescriptor {
 }
 
 export interface PropControlExtra {
+  displayMapper?: (value: string) => string;
```

Comment:
the value parameter should also support number too?

Reply:
Not sure, the display transformation only applies to the SelectOptions components since the rest of the Props are user inputs and the select is selecting from a list of options, and the SelectOption value takes strings.

So any number select option would be considered a string already right?

Reply:
ahh okayy! If its guaranteed so.